github/event_types.go

```diff
@@ -491,6 +491,7 @@ type PullRequestEvent struct {
 
 	// The following fields are only populated by Webhook events.
 	Changes            *EditChange   `json:"changes,omitempty"`
+	RequestedReviewer  *User         `json:"requested_reviewer,omitempty"`  // Populated in "review_requested", "review_request_removed" event deliveries.
```

Comment:
Where is the documentation for this field in the GitHub API docs?
I'm not finding it.

Reply:
Unfortunately, I believe that the PullRequestEvent type docs [1]  are incorrect / incomplete.

For a PullRequestEvent `review_requested` action, the docs refer to an "additional field called `requested_reviewers`". However, the field name actually returned on the event is `requested_reviewer` (singular) with a single user.

Of course, multiple users can be assigned as a reviewer at once. The full list of reviewers assigned to a PR is embedded in the PullRequest object within the event. The field name in the PullRequest object is `requested_reviewers` (plural).

I've attached the pretty-printed payload of a PullRequestEvent `review_requested` action to help illustrate this.

[1]  https://developer.github.com/v3/activity/events/types/#pullrequestevent

[review_requested_json.txt](https://github.com/google/go-github/files/1765457/review_requested_json.txt)

Reply:
If you don't mind, @stephen-soltesz, maybe you could contact support@github.com and describe to them the discrepancy in their docs and ask them to clarify or fix their API documentation?

That would be fantastic.

Reply:
Sent.

Reply:
@gmlewis, From support@:

> Thanks for reaching out and for the detailed report. Yeah, I believe you're right -- the documentation seems incorrect here. The payload will include a top-level requested_reviewer field (referencing the reviewer for this event -- the one which was added or removed), and a nested pull_request.requested_reviewer field (referencing all reviewers for the pull request). I'll open an internal issue so that the documentation can be updated.

They'll let me know when the docs are updated, but it's not possible to provide an ETA.

Is the functionality of the current PR conditional on the doc updates?

Reply:
Excellent!  Thank you, @stephen-soltesz for contacting GitHub support and reporting back... that is super helpful.

No, we don't need to wait for their documentation to update since we heard back from them, but it would be nice to keep the issue open to possibly update our docs once they do.

So I think we are good to move ahead with this PR... I'll give it another look over and it would be great to get a second LGTM before merging.

Reply:
@lyoshenka - could you please attempt to add comments here to distinguish between the singular and plural fields based on the reply from GitHub support?